File: C/D5/5 - while loop/5.cpp
// Write a C program to print the table of given number.

#include <stdio.h>

int main(){
     int i = 1, num;

     printf("Enter number: ");
     scanf("%d", &num);

     while(i<=10){
        printf("%d * %d = %d \n", num, i, (num * i));
        i++;
     }
     
}

File: Python/D6 - T/5 - palindrom num/5.py
# Write a python program to check the given number is palindrom or not.
#  for example: input = 121, reverse of input = 121. so number is palindrom.

num = int(input('Enter the number : '))
digit = 0
new_num = num
rev_num = 0

while(num != 0):
    digit = num % 10
    rev_num = (rev_num * 10) + digit
    num = num // 10
    

if (new_num == rev_num):
    print("number is palindrome")
else:
    print("number is not palindrome")
